main.py

```diff
@@ -0,0 +1,5 @@
+from gateio_new_coins_announcements_bot.main import main
+
+if __name__ == "__main__":
+    # TODO: refactor this main call
+    main()
```

Comment:
This needs to be changed, for now i just call the main method inside the main module.
I didn't want to keep the old main method here becasuse it had too many dependencies.

Reply:
I think for this PR, this is fine.
We should start addressing this in another PR/Issue, when we begin to untangle the main method into smaller pieces.

Reply:
@Linus045 yeah, for sure!